tests/components/myuplink/test_switch.py

```diff
@@ -19,6 +19,11 @@
 
 ENTITY_ID = "switch.f730_cu_3x400v_temporary_lux"
 ENTITY_FRIENDLY_NAME = "F730 CU 3x400V Tempo­rary lux"
+ENTITY_STATE = STATE_OFF
```

Comment:
This just makes it harder to read and understand the test.

Reply:
See comment below

Reply:
I still don't see a reason.

Reply:
Fair enough, my thougths were already in updating the pending test_sensor.py where it is more relevant.
I will revert.